docs/dashboard-setup/whitelisting.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Client ID and Whitelisting references
```

Comment:
There's no mention of Client ID on this page. Let's adjust the title accordingly.

The following looks good to me for sidelabel. title can be longer, like `Whitelisting your website domain or Mobile DeepLinks`, up to you.
- Whitelist
- Whitelisting